rero_ils/modules/documents/listener.py

```diff
@@ -39,7 +39,9 @@ def enrich_document_data(sender, json=None, record=None, index=None,
         es_holdings = HoldingsSearch().filter(
             'term', document__pid=document_pid
         ).scan()
+        organisation_pids = set()
         for holding in es_holdings:
+            organisation_pids.add(holding['organisation']['pid'])
```

Comment:
This variable is not used in the code. Is that normal?

Reply:
No it is from old code.